routes/assetsRoute.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const express = require('express');
+const multer = require('multer');
+const crypto = require('crypto');
+const S3 = require('../services/aws-s3');
+
+const upload = multer({ dest: 'uploads/' });
+
+const router = express.Router();
+
+/**
+ * Create a new expression
+ */
+router.post('', upload.single('image'), async (req, res) => {
+  const { file, body, user = {} } = req;
+  const { id: owner } = user;
+  const { text, levels } = body;
+
+  let image = 'hello';
+  if (file) {
```

Comment:
Not sure if this is common in js, but how do you feel about early out instead? That is, if not file, then return some sort of error, then allow yourself to do the rest of this outside of an if body?

Reply:
i like early outs. I guess i just got lazy last night

Reply:
And this?